### PR TITLE
Cgo#73

### LINKED ISSUE #73: End of Directions Screen
# Description
* When I'm finished my directions I get no feedback that I'm done

## Expected Result
* End of Directions Screen (You're Here!)

## Actual Result
* Just gets stuck on last direction with an arrow to keep going

## Steps
1. Open Application
2. Open Directions Screen
3. Select Directions
4. Go To Last Direction
5. Arrow that does nothing -->

### PR DESCRIPTION
**Pull request by danielwiktorczyk**
Fixes #73

### CODE REVIEW

File: lib/services/outdoor_path_service.dart [hunk 1]
```diff
@@ -589,4 +589,8 @@ class OutdoorPathService {
   bool isLastInstruction() {
     return _currentInstruction == _singleDirections.length - 1;
   }
+
```
Comment (DBatistaFerreira) [inline lib/services/outdoor_path_service.dart:592]: Good

File: lib/widgets/component/directions_panel.dart [hunk 2]
```diff
@@ -14,11 +15,16 @@ class DirectionsPanel extends StatefulWidget {
 }
 
 class DirectionsPanelState extends State<DirectionsPanel> {
```
Comment (DBatistaFerreira) [inline lib/widgets/component/directions_panel.dart:17]: Whats all this whitespace?

**Review by DBatistaFerreira — APPROVED** [review]
Just fix the whitespace please before merging

File: lib/widgets/component/directions_panel.dart (hunk 2 above)
Comment (danielwiktorczyk) [reply lib/widgets/component/directions_panel.dart:17]: Fixed

**Pull request merged**